src/client/components/download-files/download-files.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <ul class="download-files">
+    <li
+      v-for="item in files"
+      :key="item.id"
+      class="download-files__item">
+      <a
+        :href="item.file.url"
+        :download="`${item.slug}.${item.file.format}`"
+        class="download-files__link">{{ item.title }}</a>
+    </li>
+  </ul>
+</template>
+
+<script>
+export default {
+  props: {
+    files: {
+      type: Array,
+      required: true,
+      validator: function(data) {
+        const obj = data[0]
+        return obj.title && obj.file && obj.file.url && obj.file.format && obj.slug && obj.id
+      },
+    },
+  },
+}
+</script>
+
+<style>
+.download-files__link::after {
```

Comment:
Should we move this to `::before` so that the download icons are always vertically aligned?